src/main/java/org/sagebionetworks/web/client/widget/team/controller/TeamEditModalWidget.java

```diff
@@ -148,6 +158,20 @@ private void show() {
     view.hideLoading();
     view.clear();
     view.configure(team);
+    //SWC-6259
+    Boolean canPublicJoin = team.getCanPublicJoin();
+    Boolean canRequestMembership = team.getCanRequestMembership();
+    // default:
+    view.setTeamManagerAuthRequiredOptionActive();
+    if (canRequestMembership == null) {
+      if (canPublicJoin != null && canPublicJoin) {
```

Comment:
Style suggestion, take it or leave it

```suggestion
      if (Boolean.TRUE.equals(canPublicJoin)) {
```